book/src/keymap.md

```diff
@@ -223,6 +223,7 @@ Jumps to various locations.
 | `.`   | Go to last modification in current file          | `goto_last_modification`   |
 | `j`   | Move down textual (instead of visual) line       | `move_line_down`           |
 | `k`   | Move up textual (instead of visual) line         | `move_line_up`             |
+| `w`   | Show labels at each word and select the word that belongs to the entered labels | `goto_word` |
```

Comment:
The command is documented as `goto_word` but is actually called `goto_label`

Reply:
its actually the documentation that is right here, I renamed the command in the codebase to match